packages/core/src/builder/outputBuilder.ts

```diff
@@ -135,6 +136,12 @@ export class OutputBuilder {
     return this;
   }
 
+  public addNfts(...tokenIds: TokenId[]): OutputBuilder {
+    const tokens = tokenIds.map((tokenId) => ({ tokenId, amount: 1n }));
```

Comment:
Let's avoid compatibility issues with `bigint` literals by replacing it with `_1n` constant from `common` package.

Some parsers still not handle `bigint` literals very well 😅, so we adopted a practice to replace them with `_*n` constants in production code.

Reply:
Done :+1: & changeset added